workers/unity/Packages/io.improbable.gdk.mobile/Editor/iOSLaunchUtils.cs

```diff
@@ -77,45 +77,37 @@ public static class iOSLaunchUtils
             return availableDevices;
         }
 
-        public static void Build()
+        public static void MenuBuild(string developmentTeamId)
         {
-            if (!Directory.Exists(XCodeProjectPath))
+            try
             {
-                throw new BuildFailedException("Was not able to find an XCode project. Did you build your iOS worker?");
+                EditorUtility.DisplayProgressBar("Preparing your Mobile Client", "Building your XCode project", 0f);
+                Build(developmentTeamId);
             }
-
-            if (!TryBuildXCodeProject(string.Empty))
+            finally
             {
-                throw new BuildFailedException($"Failed to build your XCode project. Make sure you have the Command line tools for XCode (https://developer.apple.com/download/more/) installed and check the logs.");
+                EditorUtility.ClearProgressBar();
             }
         }
 
-        public static void MenuBuild(string developmentTeamId)
+        public static void Build(string developmentTeamId = "")
```

Comment:
Can you use `string.Empty` here? I can never remember

Reply:
unfortunately not :(